src/main/java/com/maddyhome/idea/vim/listener/VimListenerManager.kt

```diff
@@ -344,7 +373,9 @@ internal object VimListenerManager {
     }
 
     override fun editorReleased(event: EditorFactoryEvent) {
-      injector.markService.editorReleased(event.editor.vim)
+      val vimEditor = event.editor.vim
```

Comment:
Are we sure this function is called when the editor is released? I remember there were some details about it.

Reply:
It is definitely called when the editor is released (I've just rechecked)